lib/sqlalchemy/dialects/mysql/mysqldb.py

```diff
@@ -189,6 +189,7 @@ def create_connect_args(self, url):
         util.coerce_kw_type(opts, "write_timeout", int)
         util.coerce_kw_type(opts, "client_flag", int)
         util.coerce_kw_type(opts, "local_infile", int)
+        util.coerce_kw_type(opts, "ssl_mode", str)
```

Comment:
I just merged this and I have a doubt..."ssl_mode" does not go into the ssl dictionary?    This would indicate there was no need to add this at all, as we are already transferring all of url.query into opts.  so there's nothing "wrong" with this patch but it's unnecessary.

Reply:
Yeah, I intentionally did not put it into SSL dict, because all the values from that dict are then passed to the `mysql_ssl_set` method.  But this is a separate configurable option, which is used in `mysql_options` method with `MYSQL_OPT_SSL_MODE` key.